src/metabase/util/ssh.clj

```diff
@@ -50,6 +54,9 @@
         session                    (doto ^ClientSession (.getSession conn-status)
                                      (maybe-add-tunnel-password! tunnel-pass)
                                      (maybe-add-tunnel-private-key! tunnel-private-key tunnel-private-key-passphrase)
+                                     (.setSessionHeartbeat SessionHeartbeatController$HeartbeatType/IGNORE
+                                                           TimeUnit/SECONDS
+                                                           180)
```

Comment:
added issue to track this becoming configurable

Reply:
yeah normally at the very least it's nice to define this as a constant somewhere e.g.

```clj
(def ^:private heartbeat-interval-seconds 180)
```

which would let you override it with `with-redefs` or the like for test purposes. It's the whole "magic numbers" thing -- https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants

I think it's a minor issue tho so I don't want to block merge on this